view/theme/frio/templates/wall_thread.tpl

```diff
@@ -289,6 +208,14 @@ as the value of $top_child_total (this is done at the end of this file)
 		</div>
 		{{/if}}
 
+		<ul class="nav nav-pills preferences">
+			{{if $item.network_icon != ""}}
+                        <li><span class="wall-item-network"><i class="fa fa-{{$item.network_icon}}" title="{{$item.network_name}}" aria-hidden="true"></i></span></li>
```

Comment:
Code standards: Inconsistent indenting style.

Reply:
I replaced the spaces with tabs there.